src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -237,9 +244,11 @@ def build_bundle(
             if len(source_paths) == 1:
                 # copy a single file as the given destination path
                 symlink_or_copy(source_paths[0], dest_path)
+                created_files[source_paths[0].resolve()] = dest_path
             else:
                 # refuse to map multiple source files to one destination (undefined behaviour)
                 raise TooManyFilesError(dest_path)
+    return created_files
```

Comment:
nit: I keep getting back to this name, and something feels off. I think it's because in my mind, bundle is something that is essentially acting on a diff. So it becomes unclear here if "created" means newly added files, or all files that are supposed to be mapped. It's the latter of course, reading the code, but the name should reflect that. How about something a lot more generic like "deploy_root_file_mapping" / "deploy_root_files" / "mapped_files" etc? Might want the team's input on that one, naming is hard.

Reply:
Renamed to `mapped_files`